libctru/include/3ds/services/fspxi.h

```diff
@@ -597,7 +597,7 @@ Result FSPXI_ReadNandReport(Handle serviceHandle, void* buffer, u32 size, u32 un
  * @brief Unknown command 0x56
  * @remark Called by FSUSER_ControlArchive with ArchiveAction 0x789D
  */
-Result FSPXI_Unknown0x56(Handle serviceHandle, u32 (*out)[4], FS_Archive archive, FS_Path path);
+Result FSPXI_Unknown0x56(Handle serviceHandle, u32 out[static 4], FS_Archive archive, FS_Path path);
```

Comment:
What does that even mean/do? Never seen this syntax in the wild.

Reply:
It means that "out" cannot be NULL

Reply:
Huh, that one is new. Thanks.

Reply:
[Revert this commit, as it makes libctru incompatible with C++](https://gcc.godbolt.org/z/4kPw5M).